lib/page/set-default-values/set-default-values.js

```diff
@@ -11,8 +11,11 @@ const setInstanceDefaults = (instance) => {
 
   const schemaProps = schema.properties
   Object.keys(schemaProps).forEach(property => {
+    const schemaProperty = schemaProps[property]
+    if (schemaProperty.const !== undefined) {
```

Comment:
I wish there was a more elegant way of saying this. But I see theres not.

Reply:
heh